flanker-test-app/build.gradle

```diff
@@ -0,0 +1,88 @@
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        multiDexEnabled true
+        vectorDrawables.useSupportLibrary = true
+
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments = ['autovaluegson.mutableAdaptersWithDefaultSetters': 'true']
+            }
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        targetCompatibility 1.8
+        sourceCompatibility 1.8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    implementation project(':flanker')
+
+    implementation "org.sagebionetworks.research:mobile-ui:$sage_research_version"
+    implementation "org.sagebionetworks.research:domain:$sage_research_version"
+    implementation "org.sagebionetworks.research:presentation:$sage_research_version"
+    implementation "org.sagebionetworks.research:data:$sage_research_version"
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation 'com.google.code.gson:gson:2.8.5'
+
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+
+    // Logging
+    implementation 'org.slf4j:slf4j-api:1.7.21'
+    implementation 'com.github.tony19:logback-android-core:1.1.1-6'
+    implementation 'com.github.tony19:logback-android-classic:1.1.1-6', {
+        exclude group: 'com.google.android', module: 'android'
+    }
+    testImplementation 'org.slf4j:slf4j-simple:1.7.21'
+
+    // Dagger
+    implementation "com.google.dagger:dagger-android:$dagger_version"
+    implementation "com.google.dagger:dagger-android-support:$dagger_version"
+    kapt "com.google.dagger:dagger-android-processor:$dagger_version"
+    kapt "com.google.dagger:dagger-compiler:$dagger_version"
+    kaptTest "com.google.dagger:dagger-compiler:$dagger_version"
+
+    // Gson
+    implementation 'com.google.code.gson:gson:2.8.2'
+    implementation 'com.ryanharter.auto.value:auto-value-parcel-adapter:0.2.6'
+    implementation "com.ryanharter.auto.value:auto-value-gson-annotations:$auto_value_gson_version"
+    kapt "com.google.auto.value:auto-value:$auto_value_version"
+    kapt 'com.ryanharter.auto.value:auto-value-parcel:0.2.6'
+    kapt "com.ryanharter.auto.value:auto-value-gson:$auto_value_gson_version"
+    kapt "com.google.auto.value:auto-value-annotations:$auto_value_version" // include again for auto-value-gson
+
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'androidx.test:runner:1.1.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
+}
```

Comment:
at the end of every file should be one more line. Please add to your files that missing.
To do it automatically you can use (Settings → Editor → General → Ensure line feed at file end on Save)

Reply:
Changed it. Not sure why this is preferred over default AndroidStudio settings